reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+
+This is another reason why I think the term 'theoretical risk' is not a good choice.
+Indeed, based on the example on page 13, it seems the authors themselves confuse real theoretical risk with Type 1 error control.
+However, it is better to use terminology that encompasses Type 1 and Type 2 errors (such as 'the probability that a test will falsify a hypothesis', or even better: the *severity* of a test).
+
+> Different philosophies differ in their terminology.
+> Deborah Mayo has good reasons to differentiate Type 1 error rate from severity, as do we.
+
+Using the term 'severity of the test' would also embed the work more strongly in philosophy of science (and specifically the work of Mayo, as she builds on Popper).
+
+> This would embed us in a different philosophy of science.
+> Popper is the most famous anti-inductionist; we deliver an inductionist account.
+
+Mayo's work has been exactly on this topic, and it is a shame it has not been discussed, even though she discusses the problem of p-hacking and Type 1 error rate inflation extensively in her 2018 book.
+The authors write "The amount of knowledge researchers gain from a particular study concerning the verisimilitude of a specific theory is what we call epistemic value".
+This is an unfortunate term, as epistemic values are already used in philosophy of science to refer to qualities such as predictive accuracy, internal coherence, external consistency, unifying power, and fertility - see Elliott, K. C. (2022). Values in Science (1st ed.). Cambridge University Press. <https://doi.org/10.1017/9781009052597>.
+I would pick another terminology.
+In our own work published in Psychological Methods we called this "the marginal gain in expected utility (or usefulness) of scientific claims". (Isager, P. M., van Aert, R. C. M., Bahník, Š., Brandt, M. J., DeSoto, K. A., Giner-Sorolla, R., Krueger, J. I., Perugini, M., Ropovik, I., van 't Veer, A. E., Vranka, M., & Lakens, D. (2023).
+Deciding what to replicate: A decision model for replication study selection under resource and knowledge constraints. Psychological Methods, 28(2), 438-451. [https://doi.org/10.1037/met0000438)/](https://doi.org/10.1037/met0000438)/){.uri}.
+There are probably other existing terms for the same idea in the literature. The authors discuss P(E\|H) as 'detectability' or the probability of detecting evidence if a hypothesis holds.
+This is thus simply statistical power, or the true positive rate.
+I think the term detectability is not particularly useful, as it is more strongly associated with signal detection theory - but that is not what the authors mean with the term, as they admit the term is simply statistical power when applied to hypothesis testing (and I don't know what else it could be applied to).
+
+> Hypothesis testing and scientific inference are different things.
```

Comment:
Still, I think it would be good to discuss all suggested terms in the paper and as to how they relate to our terms.